test/fixtures/dto_fixtures.py

```diff
@@ -0,0 +1,24 @@
+from uuid import uuid4
+
+import pytest
+
+from odp_sdk.dto import MetadataDto, ResourceDto
+
+
+@pytest.fixture()
+def common_resource_dto():
```

Comment:
This returns a RAW dataset manifest. Maybe name it accordingly?

Reply:
Renamed